pkgs/tools/security/tcplay/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, cmake, pkgconfig, libgcrypt, libuuid, devicemapper, libudev, openssl, libgpgerror }:
+
+stdenv.mkDerivation rec {
+  name = "tcplay-2.0";
+
+  src = fetchurl {
```

Comment:
Oh, if I understand this correctly (and this is my first nix package, so I might as well not be) — `fetchFromGithub` checkouts the specified revision|branch from git directly.
Is it any better than downloading the official release tarball, as tcplay authors recommend to do [here](https://github.com/bwalex/tc-play#download-for-packaging)?